Editor/CoreAPI/SettingsKeys.cs

```diff
@@ -12,5 +12,12 @@ public static class SettingsKeys
         public const string DeploymentGameName = "DeploymentGameName";
         public const string DeploymentBuildFilePath = "DeploymentBuildFilePath";
         public const string DeploymentBuildFolderPath = "DeploymentBuildFolderPath";
+        public const string FleetName = "FleetName";
+        public const string FleetId = "FleetId";
+        public const string ComputeName = "ComputeName";
+        public const string IpAddress = "IpAddress";
+        public const string WebSocketUrl = "WebSocketUrl";
+        public const string SelectedProfile = "SelectedProfile";
+        public const string SelectedFleetIndex = "SelectedFleetIndex";
```

Comment:
We should avoid indexes when adding to the settings file, because that data isn't guaranteed to match when the plugin is re-opened. Also I don't think we're using this currently so we can remove for this PR

Reply:
Removed